### PR TITLE
Update GitHub templates

### PR DESCRIPTION
**Pull request by danyf90**
### Short description 📝

Try to improve templates

### Checklist ✅

- [x] The code architecture and patterns are consistent with the rest of the codebase.
- [x] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
- [x] The PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, whenever it should be included in the “Added”, “Fixed” or “Changed” section of the CHANGELOG. Note: when included in the CHANGELOG, the title of the PR will be used as entry, please make sure it is clear and suitable.
- [x] In case the PR introduces changes that affect users, the documentation has been updated.
- [x] In case the PR introduces changes that affect how the cache artifact is generated starting from the `TuistGraph.Target`, the `Constants.cacheVersion` has been updated.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_new.yml
Comment (danyf90) [inline .github/ISSUE_TEMPLATE/bug_new.yml:11]: Aligned the bug form to the Markdown one, and renamed it to be able to test it as otherwise it wasn't showing. If it works fine, we can remove the markdown

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (danyf90) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: If there is an RFC, it can be linked in the line above

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (danyf90) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Users can't set the label, so I have simplified this line

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (danyf90) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: This is enforced by the CI

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (danyf90) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: This is way to specific and IMO useless

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (kwridan) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: I think it's worth a reviewer manually testing / verifying the changes where possible. Automated tests are of course preferred, but sometimes a manual test can help spot some interesting things an automated test may have not been written for :)

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (kwridan) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: It could be useful for the the contributors / maintainers as a checklist before merging?

**Review by kwridan — APPROVED** [review]
Thanks @danyf90

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (danyf90) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: I agree, but:
- adding a checkbox wouldn't probably do much difference
- the link basically just says to run unit and integration/automation tests

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (danyf90) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: Makes sense, I'll just make clear that it's for the contributors / maintainers 👍

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (fortmarek) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: Maybe we can include the link as a comment at the very least? The link not only contains how to run tests but how we test in the codebase in general. I'd say it's good to nudge new contributors to test their PRs _and_ tell them how to test.

**Review by fortmarek — APPROVED** [review]
Small comment, but mostly good with the changes, thanks @danyf90!

**Pull request merged**
